driver-core/src/main/java/com/datastax/driver/core/querybuilder/QueryBuilder.java

```diff
@@ -351,6 +351,19 @@ public static Clause in(String name, Object... values) {
      * @param values the values
      * @return the corresponding where clause.
      */
+    public static Clause in(String name, Collection<?> values) {
```

Comment:
Should we also modify/overload the method `QueryBuilder.in(List<String> names, List<?> values)`?

Reply:
that's a cool idea.  I'll do that too

Reply:
Hrmm, there are so many places to do that.  Also for gt, lt, lte, gte.  I'm starting to think that maybe the deprecated route is not the best route since we'll create a lot of duplicate methods.  Since this change is to fix a minor inconvenience, maybe it's best to wait until java driver 4,  before adjusting the signatures, wdyt?

Reply:
I'd vote to modify the current one and the `QueryBuilder.in(List<String> names, List<?> values)` to be consistent, and the rest later indeed wdyt?